### PR TITLE
Paulov-t Readme, License and Funding link changes

### PR DESCRIPTION
**Pull request by paulov-t**
Hello, 

Hope you are all very well. 

After I have taken some time away. I have been looking through my old open source projects. This is one of the projects that caught me off guard with the amount of forks of the project, lots of which provided little to no credit together with no inferred permission. I now ensure that all my open source projects are strictly protected from forks when permission has not been provided or just stay closed source.

I have created a PR with the following corrections and clarifications. 

- LICENSE. My work was always "unlicensed", it seems people were not aware of that when making forks of the project, so this PR should make it clearer.
- README. Point to the new License document for my work.
- FUNDING. Remove me as I no longer work on this project and don't deserve any further donations for it.

Could you please look over it and let me know what you think? I might have missed a few things?? Thanks guys. Much love. <3

### CODE REVIEW

**Comment by Plootie** [comment]
It's always nice to see you around Paul and I hope you're doing well. I'll have a look around and see if I can spot anything you might have missed tonight but I did want to say something about this 👇

> * FUNDING. Remove me as I no longer work on this project and don't deserve any further donations for it.

You spent years working on this project and SIT wouldn't exist without you. I feel that even if you aren't actively contributing anymore you deserve the recognition and support. If you do really want it taken off we'll of course do so but maybe have a think over this :3

**Comment by samurailink3** [comment]
More general comment: If you want people to contact you on Discord to discuss fork-ability (which, again, is invalid), you must open up message/friend requests because right now: No one can contact you.

**Comment by gmfre** [comment]
Please consider a more permissible license for the project. Your work sets a high standard and leaving it open-sourced provides a long-term sustainability of your project for years to come.

SIT is the standard for cooperative gameplay. Please consider an open source license that has a stronger emphasis on accrediting the authors / SIT Team.

Thank you very much for your hard work. Please consider the weight of you deciding to pursue this license change.

**Review by kahvana166 — CHANGES_REQUESTED** [review]
In addition to the requested changes (which have incorrectly been marked as resolved), I would like to express my dissatisfaction at being blocked from providing further comments and clarifications regarding the license changes  on my main github account after providing the previous change requests. I believe civil discussion is possible without needlessly silencing individuals regarding the contents of the PR. If I've done anything on my main account that was not appropiate during the discussion, I would gladly see you adress publicly my misbehaviour. Otherwise I would like to see my commenting and review rights to be reinstated.

image
image

File: LICENSE-Paulov-t.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+**Paulov-t work is unlicensed**
+
+Any work or fork that is not Stay in Tarkov and uses any line of code from [Paulov-t](https://github.com/paulov-t), including but not limited to previous commits and forks of old commits is strictly prohibited. If you are using any [Paulov-t](https://github.com/paulov-t) code in your works, whether currently or previously, you can request permission directly from [Paulov-t](https://github.com/paulov-t) via GitHub, Email or Discord.
+
+**Redistribution**
+- You cannot reproduce and distribute copies of the Work (by [Paulov-t](https://github.com/paulov-t)) or Derivative Works (by [Paulov-t](https://github.com/paulov-t)) thereof in any medium, with or without modifications, and in Source or Object form.
```
Comment (kahvana166) [inline LICENSE-Paulov-t.md:12]: The license permits me to distribute in binary form and make modifications to the binary, I assume this is unintended.

File: LICENSE-Paulov-t.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Paulov-t UNLICENSED
+
+Copyright (c) 2021-2024 [Paulov-t](https://github.com/paulov-t). All rights reserved.
+
+THE SOFTWARE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+**Paulov-t work is unlicensed**
+
+Any work or fork that is not Stay in Tarkov and uses any line of code from [Paulov-t](https://github.com/paulov-t), including but not limited to previous commits and forks of old commits is strictly prohibited. If you are using any [Paulov-t](https://github.com/paulov-t) code in your works, whether currently or previously, you can request permission directly from [Paulov-t](https://github.com/paulov-t) via GitHub, Email or Discord.
```
Comment (kahvana166) [inline LICENSE-Paulov-t.md:9]: "Work" and "Fork" are not defined terms in the license. You cannot retroactively change the license, only enforce it from the point of this being merged.

File: LICENSE-Paulov-t.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+**Redistribution**
+- You cannot reproduce and distribute copies of the Work (by [Paulov-t](https://github.com/paulov-t)) or Derivative Works (by [Paulov-t](https://github.com/paulov-t)) thereof in any medium, with or without modifications, and in Source or Object form.
+
+**Expiry of old NCSA Open Source License and this replacement**: 
+-  After the project was moved out of [Paulov-t](https://github.com/paulov-t) to [Stay In Tarkov](https://github.com/stayintarkov). [Paulov-t](https://github.com/paulov-t) was incorrectly added to the original [License](https://github.com/StayInTarkov/StayInTarkov.Client/commit/a2d111eeac96b4ddb15f5fc5a182330f25ddbe4f#diff-4673a3aba01813b595de187a7a6e9e63a3491d55821606fecd9f13a10c188a1d) with the message `Paulov-t - Original Stay In Tarkov (SIT) Code`. Original Stay in Tarkov (SIT) code means code merged from the [Paulov-t GitHub](https://github.com/paulov-t) and no code developed after the fact. You are therefore accepting LICENSE any original code written up to the point of the move to Stay In Tarkov organisation.
```
Comment (kahvana166) [inline LICENSE-Paulov-t.md:15]: `LICENSE.md` up to this point was the only lawful license present in the repository, therefore any code up to this point have been covered by this license.

File: LICENSE-Paulov-t.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+- It added by Paulov that the current License is also **Unlicensed** [here](https://github.com/StayInTarkov/StayInTarkov.Client/commit/a2d111eeac96b4ddb15f5fc5a182330f25ddbe4f#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5) but also suggests that `None of my own work is Licensed. This is solely a just for fun project. I don't care what you do with it`. This is a poorly formatted phrase that does not grant a license.
```
Comment (kahvana166) [inline LICENSE-Paulov-t.md:16]: Indeed correct, and confirms that LICENSE.md is the only lawful license present.

File: LICENSE-Paulov-t.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+- The README.md and therefore agreement was changed [here](https://github.com/StayInTarkov/StayInTarkov.Client/commit/638dea7394741496b617900a685a4f449f49b318). The LICENSE.md at that time did not and does not reflect any code written by [Paulov-t](https://github.com/paulov-t) for the Stay in Tarkov organisation. Key segment here is that LICENSE.md is not the only section that defined the License agreement. If you forked Stay in Tarkov on or after this commit, you are in fact accepting several LICENSE agreements that clashed and were not clear. All code by [Paulov-t](https://github.com/paulov-t) from this point is therefore UNLICENSED. See [GitHub Licensing](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository#choosing-the-right-license) for more information `without a license, the default copyright laws apply, meaning that you retain all rights to your source code and no one may reproduce, distribute, or create derivative works from your work`
```
Comment (kahvana166) [inline LICENSE-Paulov-t.md:17]: Incorrect. Paulov-t's code is also under NCSA coverage as it was the only lawful license present.

File: LICENSE-Paulov-t.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+- In the interest of openness and simplicity. If you forked Stay in Tarkov at any time AFTER it was moved to the organisation [Stay In Tarkov](https://github.com/stayintarkov). It is recommended you get in touch and/or at least properly credit [Paulov-t](https://github.com/paulov-t) in all code written by them and releases of code that derives from this work. This will cover you and your project. Please remember `without a license, the default copyright laws apply, meaning that you retain all rights to your source code and no one may reproduce, distribute, or create derivative works from your work` even with `None of my own work is Licensed. This is solely a just for fun project. I don't care what you do with it` can be argued as conflicting of terms.
```
Comment (kahvana166) [inline LICENSE-Paulov-t.md:18]: Incorrect, a license was present (LICENSE.md) an in effect.

**Review by kahvana166 — CHANGES_REQUESTED** [review]
The previous changes have not been resolved with the most recent commit, tt however introduced more issues regarding the legality of the license. I hope these can be resolved in an appropiate manner.

**Comment by samurailink3** [comment]
I'm still confused by your intent with these changes. The previously-committed license seemed to target SIT forks and make forking the project harder (if not outright impossible) and attempted to target already-existing forks. The current licensing changes don't seem to do much except cause confusion.

What is your intent with these changes? What are you trying to accomplish?

**Comment by nexus4880** [comment]
> I have explained the reason for this PR. I have explained that the current (NCSA, MIT and Unlicense) license agreements defined in 3 places are misleading, incorrect and need rectifying. My changes are in for review by the current Stay in Tarkov team who are yet to make any comments apart from Ploot. I have no influence over whether this is accepted.

You may not have influence but you can fix this by recognizing this is nonsense and closing the PR.

Admittedly the licensing on the project is questionable at best.

To better understand what I'm about to say I'd like to make it clear that when acquiring open source code you are acquiring the code on the license that you agree to it on. If you acquire code under X and the manager of the repository changes the license to Y the code that you have is still permissible under X.

Now that we've gotten that out of the way if a project is forked prior to commit `638dea7394741496b617900a685a4f449f49b318` then the following applies.

According to [the license in the repo](https://github.com/stayintarkov/StayInTarkov.Client/blob/638dea7394741496b617900a685a4f449f49b318/LICENSE.md) the project is NCSA.

You claim your work is "unlicensed" which according to GitHub [you retain all rights to your source code and no one may reproduce, distribute, or create derivative works from your work](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository).

So while this would normally mean we can't use your code as you retain all rights to it... [someone cannot legally use any part of your GitHub project in their code, even if it's public, **unless you explicitly give them the right to do so**.](https://opensource.guide/legal/#are-public-github-projects-open-source)

[You have done this on the commit from which that project would be based on.](https://github.com/stayintarkov/StayInTarkov.Client/blob/638dea7394741496b617900a685a4f449f49b318/README.md?plain=1#L190)

Changing the NCSA license would require consent from all (including former) contributors.

[If you publish your source code in a public repository on GitHub, according to the Terms of Service, other users of GitHub.com have the right to view and fork your repository.](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository)

**Comment by mihaicm93** [comment]
> You may not have influence but you can fix this by recognizing this is nonsense and closing the PR.
> 
> Admittedly the licensing on the project is questionable at best.
The entire licensing of SIT is a shitshow because nobody really thought this would happen.  I'll make sure I never make the same mistake of putting in all this work just to get screwed over by license crap and people that "help" when it's convinent for them after wich _they take your toys and go lock themselfs in their "home" to play with them_. What exactly is nonsense in him adding this ? I want to know as this seems like did touch a lot of people the wrong way.

> 
> To better understand what I'm about to say I'd like to make it clear that when acquiring open source code you are acquiring the code on the license that you agree to it on. If you acquire code under X and the manager of the repository changes the license to Y the code that you have is still permissible under X.
> 
> Now that we've gotten that out of the way if a project is forked prior to commit `638dea7394741496b617900a685a4f449f49b318` then the following applies.
> 
> According to [the license in the repo](https://github.com/stayintarkov/StayInTarkov.Client/blob/638dea7394741496b617900a685a4f449f49b318/LICENSE.md) the project is NCSA.
> 
> You claim your work is "unlicensed" which according to GitHub [you retain all rights to your source code and no one may reproduce, distribute, or create derivative works from your work](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository).
> 
> So while this would normally mean we can't use your code as you retain all rights to it... [someone cannot legally use any part of your GitHub project in their code, even if it's public, **unless you explicitly give them the right to do so**.](https://opensource.guide/legal/#are-public-github-projects-open-source)
> 
> [You have done this on the commit from which that project would be based on.](https://github.com/stayintarkov/StayInTarkov.Client/blob/638dea7394741496b617900a685a4f449f49b318/README.md?plain=1#L190)
> 
@paulov-t work was always unlicensed.


> Changing the NCSA license would require consent from all (including former) contributors.
> 
> [If you publish your source code in a public repository on GitHub, according to the Terms of Service, other users of GitHub.com have the right to view and fork your repository.](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service#5-license-grant-to-other-users)

It is not a change on the project licence, but on his work done on the project, am I missing something ?

**Comment by mihaicm93** [comment]
> At last, what caught me off guard:
> 
> > his is one of the projects that caught me off guard with the amount of forks of the project, lots of which provided little to no credit together with no inferred permission.
> 
> ...didn't sit well with me and can be proven not be the case. Take a look at the repositories in the dependency graph on github (which is all the forks of this project) and inspect the repositories. These are all the `LICENSE.md` files of each repository in the dependency graph:
> 
> * https://github.com/paulov-t/StayInTarkov.Client/blob/master/LICENSE.md?plain=1#L6
> * https://github.com/mihaicm93/StayInTarkov.Client/blob/master/LICENSE.md?plain=1#L6
> * https://github.com/JimWails/StayInTarkov.Client/blob/master/LICENSE.md?plain=1#L6
> * https://github.com/DRDRDRRDDRDR/StayInTarkov.Client/blob/master/LICENSE.md?plain=1#L6
> * https://github.com/kapdon/StayInTarkov.Client/blob/master/LICENSE.md?plain=1#L6
> * https://github.com/cocorico8/StayInTarkov.Client/blob/master/LICENSE.md?plain=1#L6
> * https://github.com/belettee/StayInTarkov.Client/blob/master/LICENSE.md?plain=1#L6
> * https://github.com/yhl452493373/StayInTarkov.Client/blob/master/LICENSE.md?plain=1#L6
> * https://github.com/StuckInLimbo/StayInTarkov.Client/blob/master/LICENSE.md?plain=1#L6
> * https://github.com/hickorysb/StayInTarkov.Client/blob/master/LICENSE.md?plain=1#L6
> * https://github.com/NizZ8/StayInTarkov.Client/blob/master/LICENSE.md?plain=1#L6
> * https://github.com/MiKueYu/StayInTarkov.Client/blob/master/LICENSE.md?plain=1#L6
> * https://github.com/KamiliaBlow/StayInTarkov.Client/blob/master/LICENSE.md?plain=1#L6
> * https://github.com/Plootie/StayInTarkov.Client/blob/master/LICENSE.md?plain=1#L6
> * https://github.com/Beat2er/StayInTarkov.Client/blob/master/LICENSE.md?plain=1#L6
> * https://github.com/mikeyybroo/StayInTarkov.Client/blob/master/LICENSE.md?plain=1#L6
> * https://github.com/shibdib/StayInTarkov.Client/blob/master/LICENSE.md?plain=1#L6
> * https://github.com/BrianBoundy/StayInTarkov.Client/blob/master/LICENSE.md?plain=1#L6
> * https://github.com/trippyone/StayInTarkov.Client/blob/master/LICENSE.md?plain=1#L6
> * https://github.com/SiulSC/StayInTarkov.Client/blob/master/LICENSE.md?plain=1#L6
> 
> And from the repository that I know outside of the dependency graph containing this work in full:
> 
> * https://github.com/Neko17awa/StayInTarkov.Client-Assembly-0.14/blob/master/LICENSE.md?plain=1#L6
> 
> And from the respository that I know of outside of the dependency graph containing snippets of this project's code:
> 
> * https://github.com/project-fika/Fika-Plugin/blob/main/LICENSE-SIT.md?plain=1#L6
> 
> As you can see, all of them provide attribution in the appropiate manner. I wonder which the _lots_ of repositories are that do not provide proper attribution. Also makes one wonder what this PR really is for.

While the client mod Licence is a mess, none of those projects have any right to use the [https://github.com/stayintarkov/SIT.Aki-Server-Mod]SIT.Aki-Server-Mod [https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository]( Without a license, the default copyright laws apply, meaning that you retain all rights to your source code and no one may reproduce, distribute, or create derivative works from your work.)

**Comment by SlejmUr** [comment]
> While the client mod Licence is a mess, none of those projects have any right to use the [https://github.com/stayintarkov/SIT.Aki-Server-Mod](SIT.Aki-Server-Mod)

Yes, that is right.
> the default copyright laws apply, meaning that you retain all rights to your source code and no one may reproduce, distribute, or create derivative works from your work. 

That means no-one exluding paulov are able to edit the source code and distribute, which we all already broke. (Who forked, contributed)
Reproduce: 
Cannot make PR for the repository, nor share the code to anyone privately or openly.
Create derivative works:
Cannot make a Fork or anything that close to the code.

Also see the package.json it is MIT:
https://github.com/stayintarkov/SIT.Aki-Server-Mod/tree/26d9b7abadf96d8a056d31d9d0ecd5a817d0d11e

So what license we using?

So paulov gonna rollback into version before anyone in old repo pushed any change?
Inluding you, me and all contributors?

> The entire licensing of SIT is a shitshow because nobody really thought this would happen

It was that before, it will be that forever.
And because the project is publicly open and many mirrors, fork exist they can see what happens in the project.
You and Me known to mirror this project to our own server.
And I will NOT cease and destroy copies from my server which has a git history. 

Please consult a lawyer and it will help you. 
(should have been done in the first place where the repo got private then reuploaded)

Also changing a License you have to contact all contributors and ask for their permission, or remove all their code.
Which you can find in here: https://github.com/stayintarkov/StayInTarkov.Client/commits/master/ and clicking the All Users button.

Including:
OkaMoez, langhillie, ezra-sg, mikeyybroo, shibdib, cocorico8, debisuchan, evileyecc1, kapdon, lukehemmin, Plootie, privateryann1, Starkrights, SlejmUr, yhl452493373, ArchangelWTF, ChewieDM, TehFl0w, devbence, belettee, Lacyway, JimWails, trippyone, donunai2333, mihaicm93, paulov-t

**Comment by ArchangelWTF** [comment]
> Also changing a License you have to contact all contributors and ask for their permission, or remove all their code. Which you can find in here: https://github.com/stayintarkov/StayInTarkov.Client/commits/master/ and clicking the All Users button.
> 
> Including: OkaMoez, langhillie, ezra-sg, mikeyybroo, shibdib, cocorico8, debisuchan, evileyecc1, kapdon, lukehemmin, Plootie, privateryann1, Starkrights, SlejmUr, yhl452493373, ArchangelWTF, ChewieDM, TehFl0w, devbence, belettee, Lacyway, JimWails, trippyone, donunai2333, mihaicm93, paulov-t

I will already add it here just so that notice has already been given, if this PR is merged please consider removing my code from SIT.

**Pull request closed without merging**